lib/driver.js

```diff
@@ -945,6 +945,13 @@ export class XCUITestDriver extends BaseDriver {
           await markSystemFilesForCleanup(this.wda);
         }
 
+        // We don't restrict the version, but show what version of WDA is running on the device for debugging purposes.
+        if (this.cachedWdaStatus?.build) {
+          this.log.info(`Running WebDriverAgent version is '${this.cachedWdaStatus.build.version}'.`);
+        } else {
+          this.log.info(`Running WebDriverAgent does not provide version information. Old or customized WebDriverAgent is running on the device.`);
```

Comment:
WebDriverAgent does not provide any version information. This might indicate either a custom or an outdated build.